lib/prompt.js

```diff
@@ -183,11 +183,11 @@ class Prompt extends Events {
   }
 
   async skip() {
-    this.skipped = this.options.skip === true;
```

Comment:
why remove `this.skipped`? The point of this is to be able to see if a prompt was skipped in custom code.

Reply:
Just removed `this.skipped` as it wasn't used anywhere else in the code. I didn't think of using it in custom prompts.